### PR TITLE
Fix 3.2 release window

### PR DESCRIPTION
**Pull request by cloud-fan**
The 3.1 was released on Mar 2, so the target release day for 3.2 should be September 2 according to the 6 months release cycle.

Then the RC stage should start one month before the target release day, which is August.

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @dongjoon-hyun @gatorsmile @srowen

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
Technically, 
- Apache Spark 3.0.0 was released on June 18, 2020.
- Apache Spark 3.1.1 is delayed according to @gatorsmile 's request.

I don't think @gatorsmile meant to delay all the subsequent Apache Spark releases. We manage branches to isolate the releases. The master branch has been prepared for Apache Spark 3.2 without being blocked by Apache Spark 3.1.

**Comment by dongjoon-hyun** [comment]
Shall we discuss on the dev emailing list about this revision? As you know, I'll not be against on the dev mailing list decision. :)

**Comment by dongjoon-hyun** [comment]
Whatever we choose, just FYI, cc @dbtsai , @holdenk , @viirya , @sunchao, @attilapiros

**Comment by cloud-fan** [comment]
I'm just following the existing document
```
In general, feature ("minor") releases occur about every 6 months. Hence, Spark 2.3.0 would
generally be released about 6 months after 2.2.0.
```

I don't see any place mentioning that the 6 months is counted after the scheduled release date, or the branch cut date.

> The master branch has been prepared for Apache Spark 3.2 without being blocked by Apache Spark 3.1.

Before 3.1 was released, I think it was still the QA period and part of the community was not working on 3.2 development. I think we should have a full 6 months of community development for the new release.

Looking at the history, 2.2.0 was released 7 months after 2.1.0, 2.3.0 was released 7 months after 2.2.0. We don't have the tradition to shorten the release window of the next release if the previous release was delayed.

**Comment by dongjoon-hyun** [comment]
It would be greatly helpful if you share it to the dev mailing list to finalize this timeline.

**Comment by cloud-fan** [comment]
I've added a reply in the 3.2 thread in the dev list. I'll merge this PR after a few days if there is no objection.

**Review by HyukjinKwon — APPROVED** [review]
I'm good with this.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.
Thank you, @cloud-fan .

**Review by viirya — APPROVED** [review]
looks fine.

**Review by yaooqinn — APPROVED** [review]

**Review by attilapiros — APPROVED** [review]
LGTM

**Review by maropu — APPROVED** [review]

**Review by HeartSaVioR — APPROVED** [review]
+1 Looks OK.

**Pull request closed without merging**
